release_notes/gitops-release-notes.adoc

```diff
@@ -25,6 +25,18 @@ include::modules/making-open-source-more-inclusive.adoc[leveloffset=+1]
 
 // Modules included, most to least recent
 
+<<<<<<< HEAD
+=======
+// Release notes for Red Hat OpenShift GitOps 1.11.1
+include::modules/gitops-release-notes-1-11-1.adoc[leveloffset=+1]
+
+// Release notes for Red Hat OpenShift GitOps 1.11.0
+include::modules/gitops-release-notes-1-11-0.adoc[leveloffset=+1]
```

Comment:
@Dhruv-Soni11 L28 to L34 - These must be removed.

Reply:
Ok, I have fixed this error.